docs/background/velocities.rst

```diff
@@ -0,0 +1,521 @@
+:math:`0^\circ \leq \xi < 180^\circ`. Because the angle :math:`\xi` is
+restricted to this range, it technically refers to the position angle of the
+*eastern* half of the line of lensed images.
+
+.. image:: images/sky-plane-mu.svg
+    :height: 250
+    :align: center
+    :alt: sky plane with screen and proper motion vector
+
+
+The pulsar's orbital motion -- circular orbit
+=============================================
+
+Let's first consider the case of a pulsar in a circular orbit.
+
+
+The :math:`xyz` coordinate system and the orbital inclination
+-------------------------------------------------------------
+
+We introduce the right-handed orthonormal triad :math:`(\hat{x}, \hat{y},
+\hat{z})` linked to the binary's barycentre, with :math:`\hat{x}` and
+:math:`\hat{y}` in the plane of the sky, :math:`\hat{x}` pointing to the
+pulsar's ascending node (the point on the orbit that intersects the sky-plane,
+where the pulsar moves away from the observer), and :math:`\hat{z}` pointing
+away from the observer. Since this is a right-handed coordinate system,
+if :math:`\hat{y}` points upward as viewed by the observer, then
+:math:`\hat{x}` points to the left (and the :math:`x` coordinate increases
+towards the left).
+
+The orbital inclination is parameterised by the angle :math:`i_\mathrm{p}`
+between the direction towards the observer :math:`-\hat{z}` and the binary's
+orbital specific angular momentum
+:math:`\vec{h}_\mathrm{p} = \vec{r}_\mathrm{p} \times \vec{v}_\mathrm{p}`,
+where :math:`\vec{r}_\mathrm{p}` and :math:`\vec{v}_\mathrm{p}` denote the
+pulsar's position and velocity, respectively. This angle is naturally
+restricted to the range :math:`0^\circ \le i_\mathrm{p} < 180^\circ`. As per
+the standard convention for orbits outside the Solar System, inclinations of
+:math:`i_\mathrm{p} < 90^\circ` correspond to counterclockwise rotation on the
+sky and inclinations of :math:`i_\mathrm{p} > 90^\circ` correspond to clockwise
+rotation on the sky, with :math:`i_\mathrm{p} = 90^\circ` for an edge-on orbit
+(:math:`\vec{h}_\mathrm{p}` anti-parallel to :math:`\hat{y}`).
+
+.. container:: align-center
+
+    .. image:: images/xy-plane-orb.svg
+        :height: 250
+        :alt: xy plane with orbit
+
+    .. image:: images/yz-plane-orb.svg
+        :height: 250
+        :alt: yz plane with orbit and inclination
+
+    **Left:** observer's view, looking in the direction of :math:`\hat{z}`.
+    **Right:** side view, looking in the direction of :math:`\hat{x}`.
+
+
+The pulsar's position and velocity in :math:`xyz` coordinates
+-------------------------------------------------------------
+
+In this :math:`xyz` coordinate system,
+the pulsar's position and velocity as function of the its orbital phase
+:math:`\phi_\mathrm{p} = 2 \pi ( t - t_\mathrm{asc,p} ) / P_\mathrm{orb,p}`,
+measured from the ascending node of the pulsar's orbit, are given by
+
+.. math::
+
+    \vec{r}_\mathrm{p} &= a_\mathrm{p}
+        \left[   \cos( \phi_\mathrm{p} ),
```

Comment:
I was too lazy myself, but it might be nice to typeset this as proper vections, using `\left(\begin{array}x\\y\\z\end{array}\right)` (hope I remember that right...)

Reply:
Done.